std_msgs/Header header

#ID for Apriltag providing pose update from map lookup.
int32 apriltag_id

#Pose of camera in map coordinates after lookup.
geometry_msgs/PoseStamped map_lookup_pose